tests/components/icloud/test_config_flow.py

```diff
@@ -282,10 +290,11 @@ def init_config_flow(hass: HomeAssistantType):
     assert result["errors"] == {CONF_TRUSTED_DEVICE: "send_verification_code"}
 
 
-async def test_verification_code(hass: HomeAssistantType):
+async def test_verification_code(hass: HomeAssistantType, service: MagicMock):
     """Test verification_code step."""
     flow = init_config_flow(hass)
     await flow.async_step_user({CONF_USERNAME: USERNAME, CONF_PASSWORD: PASSWORD})
+    await flow.async_step_trusted_device({CONF_TRUSTED_DEVICE: 0})
```

Comment:
Here use `hass.config_entries.flow.async_configure`.

https://github.com/home-assistant/home-assistant/blob/8fcd0e9a7901493f89d10c0b5de6d635f7c07fe4/homeassistant/data_entry_flow.py#L116-L118

Reply:
For `test_verification_code_success` and `test_validate_verification_code_failed`, I am getting None at user_input in `IcloudFlowHandler.async_step_verification_code()`

Reply:
Don't know why since the trusted_device step is working fine.